### PR TITLE
feat: add commisery GitHub action

### LINKED ISSUE #14: [Infrastructure]: Enable Conventional Commit Message Checker
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Description

Enable https://github.com/KevinDeJong-TomTom/commisery-action for all enarx org repositories.

### PR DESCRIPTION
**Pull request by platten**
Closes #14

### CODE REVIEW

**Review by rvolosatovs — DISMISSED** [review]

File: .github/workflows/commisery.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-latest
+    steps:       
+    - name: Check-out the repo under $GITHUB_WORKSPACE
+      uses: actions/checkout@v2
+
+    - name: Run Commisery
+      uses: enarx/commisery-action@master
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        pull_request: ${{ github.event.number }}
```
Comment (npmccallum) [inline .github/workflows/commisery.yml:32]: Why does this action require our `GITHUB_TOKEN`? What permissions are we granting to it?

Comment (npmccallum) [reply .github/workflows/commisery.yml:32]: That is unnecessary for a public repo. You should submit a PR to make the token optional.

Comment (platten) [reply .github/workflows/commisery.yml:32]: This is unfortunately unfeasible, as this requires putting in a PR for the GitHub Python API library to not require a `GITHUB_TOKEN`. What I think is a better alternative is to simply restrict the access which `GITHUB_TOKEN` provides via: https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

**Comment by dpal** [comment]
This PR can be closed since the issue was addressed later by a different PR and we have shared commisery workflow already implemented.

**Pull request closed without merging**